client/common/file.c

```diff
@@ -199,143 +199,146 @@ struct rdp_file
  *
  */
 
-static int freerdp_client_rdp_file_set_integer(rdpFile* file, const char* name, int value,
-                                               int index)
+static BOOL freerdp_client_rdp_file_set_integer(rdpFile* file, const char* name, long value,
+											   SSIZE_T index)
 {
-	int standard = 0;
+	BOOL standard = FALSE;
```

Comment:
The name of this var is quite misleading as it does exactly the opposite ^^

Reply:
Ok, fixed that (including description)